scripts/clean.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+rm -rf .build-*
```

Comment:
```suggestion
rm -rf .build*
```
Else main build is not cleared

Reply:
Good catch! Added main later, will update :+1: